safety/scan/constants.py

```diff
@@ -83,6 +83,7 @@
     "[bold]Example: safety scan --output json[/bold]"
 SCAN_SAVE_AS_HELP = "In addition to regular output save the scan results to a json, html, text, or spdx file using: FORMAT FILE_PATH\n\n" \
     "[bold]Example: safety scan --save-as json results.json[/bold]"
+SCAN_CVE_HELP = "Include CVE details in the output."
```

Comment:
Just for consistency.

SCAN_CVE_HELP = "Include CVE details in the output\n\n" \
    "[bold]Example: safety scan --cve[/bold]"

Reply:
awesome